yelp_package/dockerfiles/gunicorn_exporter-sidecar/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM docker-dev.yelpcorp.com/focal_yelp
```

Comment:
Could this use `jammy`, or does that not work?

Reply:
Updated

Reply:
Thanks!